src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.hpp

```diff
@@ -239,6 +239,14 @@
                         int opc, BasicType bt, uint vector_length,
                         VectorMask vm = Assembler::unmasked);
 
+  void reduce_mul_integral_v(Register dst, Register src1, VectorRegister src2,
+                             VectorRegister vtmp1, VectorRegister vtmp2, BasicType bt,
+                             uint len, VectorMask vm = Assembler::unmasked);
```

Comment:
BTW: Do you mind renaming this param `len` to `vector_length`? Then it will be consistent with friends `compare_integral_v`, `compare_fp_v`, etc in param naming.

Reply:
sure, done.